libs/common/src/dirt/services/phishing-detection-settings.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Observable } from "rxjs";
+import { map } from "rxjs/operators";
+
+import { UserId } from "@bitwarden/user-core";
+
+import {
+  ActiveUserState,
+  PHISHING_DETECTION_DISK,
+  StateProvider,
+  UserKeyDefinition,
+} from "../../platform/state";
+
+const ENABLE_PHISHING_DETECTION = new UserKeyDefinition(
+  PHISHING_DETECTION_DISK,
+  "enablePhishingDetection",
+  {
+    deserializer: (value: boolean) => value ?? true, // Default: enabled
+    clearOn: ["logout"],
+  },
```

Comment:
We want the setting value to persist in local storage even if user logs out.
Relevant acceptance criteria from jira ticket: "Setting persists across sessions"

["An empty array should be used if the state should not ever be removed (e.g. for settings)."](https://github.com/bitwarden/clients/blob/dab1a37bfe4b3a51aa7426533fa52795b2371474/libs/state/README.md?plain=1#L140C304-L140C394)

```suggestion
const ENABLE_PHISHING_DETECTION = new UserKeyDefinition(
  PHISHING_DETECTION_DISK,
  "enablePhishingDetection",
  {
    deserializer: (value: boolean) => value ?? true, // Default: enabled
    clearOn: [],
  },
```

@Banrion